addons/packages/fluxcd-kustomize-controller/0.21.1/README.md

```diff
@@ -0,0 +1,208 @@
+      | Getting package metadata for 'kustomize-controller.fluxcd.community.tanzu.vmware.com'
+      | Creating service account 'fluxcd-kustomize-controller-default-sa'
+      | Creating cluster admin role 'fluxcd-kustomize-controller-default-cluster-role'
+      | Creating cluster role binding 'fluxcd-kustomize-controller-default-cluster-rolebinding'
+      | Creating package resource
+      / Waiting for 'PackageInstall' reconciliation for 'fluxcd-kustomize-controller'
+      \ 'PackageInstall' resource install status: Reconciling
+
+      Added installed package 'fluxcd-kustomize-controller'
+      ```
+
+   2. Verify the package install by running:
+
+       ```shell
+       tanzu package installed get fluxcd-kustomize-controller
+       ```
+
+       For example:
+
+       ```shell
+      - Retrieving installation details for fluxcd-kustomize-controller...
+        NAME:                    fluxcd-kustomize-controller
+        PACKAGE-NAME:            kustomize-controller.fluxcd.community.tanzu.vmware.com
+        PACKAGE-VERSION:         0.21.1
+        STATUS:                  Reconcile succeeded
+        CONDITIONS:              [{ReconcileSucceeded True  }]
+       ```
+
+       Verify that `STATUS` is `Reconcile succeeded`
+
+       ```shell
+       kubectl get pods -n kustomize-system
+       ```
+
+       For example:
+
+       ```shell
+       kubectl get pods -n kustomize-system
+       NAME                                    READY   STATUS    RESTARTS   AGE
+       kustomize-controller-7d96857bfc-hcbmk   1/1     Running   0          3m36s
+       ```
+
+       Verify that `STATUS` is `Running`
+
+## Try kustomize-controller
+
+1. Verify all the objects are installed:
+
+    This package would create a new namespace where all the elements of fluxcd will be hosted called `kustomize-system`
+
+    you can verify the main components of `kustomize-controller` were installed by running:
+
+    ```shell
+    kubectl get all -n kustomize-system  
+    NAME                                        READY   STATUS    RESTARTS   AGE
+    pod/kustomize-controller-7d96857bfc-hcbmk   1/1     Running   0          6m12s
+
+    NAME                                   READY   UP-TO-DATE   AVAILABLE   AGE
+    deployment.apps/kustomize-controller   1/1     1            1           6m12s
+
+    NAME                                              DESIRED   CURRENT   READY   AGE
+    replicaset.apps/kustomize-controller-7d96857bfc   1         1         1       6m12s
+    ```
+
+    you should get something really similar!
+
+2. Verify all the CRD were installed correctly:
+
+    The way you would communicate with `kustomize-controller` would be through its CRDs, this will be your main action point.
+
+    In order to check all the CRDs were installed you can run:
+
+    ```shell
+    kustomize.kustomize.toolkit.fluxcd.io                                     2022-04-10T16:12:13Z
+    ```
```

Comment:
I don't understand this. It appears to missing the command and this is just the response? Can you clarify this?